apis/gateway/v2alpha1/apirule_types.go

```diff
@@ -200,7 +200,7 @@ type JwtHeader struct {
 // ExtAuth contains configuration for paths that use external authorization.
 type ExtAuth struct {
 	// Specifies the name of the external authorization handler.
-	// +optional
+	// +kubebuilder:validation:MinItems=1
```

Comment:
I think we need to update the docs, because this is now required: https://github.com/kyma-project/api-gateway/blob/c22be59936100fa6095148e13ce761b2df387ac7/docs/user/custom-resources/apirule/v2alpha1/04-10-apirule-custom-resource.md?plain=1#L51